.github/workflows/pythonapp.yml

```diff
@@ -167,23 +161,24 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
       - name: Install dependencies
-        run: sudo apt-get install -y libopenblas-dev liblapack-dev ninja-build
+        run: sudo apt-get install -y libopenblas-dev liblapack-dev
 
-      - name: Install CI dependencies
+      - name: Install Basix C++ library
         run: |
-          pip install pytest sympy numba scipy matplotlib pytest-xdist pybind11
-          pip install git+https://github.com/FEniCS/ufl.git
-
-      - name: Install Basix
-        run: |
-          cmake -DCMAKE_BUILD_TYPE=Release -DDOWNLOAD_XTENSOR_LIBS=ON -B build-dir -S .
+          cd cpp
+          cmake -DCMAKE_BUILD_TYPE=Release -B build-dir -S . # Use make (not ninja)
```

Comment:
Why do we not want to use `ninja`?

Reply:
Good to do a build with the make generator, used frequently on HPCs (I know I don't bother installing ninja!). I left a comment there to note that it's not a mistake.